ember-headlessui/addon/components/listbox.js

```diff
@@ -181,15 +187,27 @@ export default class ListboxComponent extends Component {
     optionElement.setAttribute('data-index', this.optionElements.length - 1);
 
     if (this.args.value) {
-      if (this.args.value === optionComponent.args.value) {
-        this.selectedOptionIndex = this.activeOptionIndex =
-          this.optionElements.length - 1;
+      let isSelected;
+
+      if (this.args.multiple) {
+        isSelected = this.args.value.includes(optionComponent.args.value);
+      } else {
+        isSelected = this.args.value === optionComponent.args.value;
+      }
+
+      if (isSelected) {
+        this.selectedOptionIndexes.add(this.optionElements.length - 1);
+        this.activeOptionIndex = this.optionElements.length - 1;
 
         this.scrollIntoView(optionElement);
       }
     }
 
-    if (!this.selectedOptionIndex) {
+    scheduleOnce('afterRender', this, this.setDefaultActiveOption);
```

Comment:
can we avoid hacking runloop usage?

runloop is kinda anti-patterny and causes performance problems